front/pages/api/login.ts

```diff
@@ -400,7 +399,7 @@ async function handler(
 
       // Delete newly created user if SSO is mandatory.
       if (userCreated) {
-        await deleteUser(user);
+        await user.unsafeDelete();
```

Comment:
I'm not super happy with that `unsafeDelete()` but I don't think I can get an Authenticator here as we don't seem to always know the workspaceId